packages/libs/eda/src/lib/map/analysis/mapTypes/MapTypeHeaderStudyDetails.tsx

```diff
@@ -0,0 +1,125 @@
+import { useQuery } from '@tanstack/react-query';
+import { Tooltip } from '@veupathdb/components/lib/components/widgets/Tooltip';
+import {
+  Filter,
+  useStudyEntities,
+  useStudyMetadata,
+  useSubsettingClient,
+} from '../../../core';
+
+interface Props {
+  /** All filters applied to the map, including:
+   * - subset
+   * - marker config
+   * - timeline filters
+   * - viewport
+   */
+  filters?: Filter[];
+
+  filtersIncludingViewport: Filter[];
+
+  /** Entity space of map markers, overlay, etc */
+  outputEntityId: string;
+  totalEntityCount?: number;
+  totalEntityInSubsetCount?: number;
+  visibleEntityCount?: number;
+  onShowStudies: (showStudies: boolean) => void;
+}
+
+const { format } = new Intl.NumberFormat('en-us');
+
+export function MapTypeHeaderStudyDetails(props: Props) {
+  const {
+    filters,
+    filtersIncludingViewport,
+    outputEntityId,
+    visibleEntityCount,
+    onShowStudies,
+  } = props;
+  const { rootEntity } = useStudyMetadata();
+  const rootEntityCount = useEntityCount(
+    rootEntity.id, // Study entity
+    filtersIncludingViewport // Should be subset, map type, timeline, and viewport filters
+  );
+
+  const outputEntity = useStudyEntities().find(
+    (entity) => entity.id === outputEntityId
+  );
+
+  if (
+    rootEntityCount.data == null ||
+    visibleEntityCount == null ||
+    outputEntity == null
+  )
+    return null;
+
+  const totalCountFormatted = format(visibleEntityCount);
+  const totalRootEntityFromatted = format(rootEntityCount.data.count);
+
+  const tooltipContent = (
+    <div>
+      <p>
+        {totalCountFormatted} {outputEntity.displayNamePlural} are currently
+        visualized on the map using markers. These are the{' '}
+        {outputEntity.displayNamePlural} that
+        <ul>
+          <li>
+            satisfy all your filters (you currently have{' '}
+            {filters?.length ?? 'no'} filters active)
+          </li>
+          <li>to do: also time slider filter</li>
+          <li>
+            have data for the variable that is currently displayed on the
+            markers
+          </li>
+          <li>
+            have appropriate values for if the marker has been custom-configured
+          </li>
+        </ul>
+      </p>
+      <p>
+        The visualized data comes from {totalRootEntityFromatted}{' '}
+        {rootEntity.displayNamePlural}{' '}
+        <button type="button" onClick={() => onShowStudies(true)}>
+          Show list
+        </button>
+      </p>
+    </div>
+  );
+
+  return (
+    <>
+      <div
+        css={{
+          padding: '1em',
+          fontSize: '1.2em',
+          display: 'flex',
+          gap: '1ex',
+          alignItems: 'center',
+        }}
+      >
+        {format(visibleEntityCount)} {outputEntity.displayNamePlural} from{' '}
+        {format(rootEntityCount.data.count)} {rootEntity.displayNamePlural}
+        <Tooltip title={tooltipContent} interactive style={{ width: 'auto' }}>
```

Comment:
The tooltip is rendering half off-screen unless my browser (Firefox) window is >1800 px wide. This means the "show list" button isn't visible. I'm not sure why, sorry!

Reply:
This is because the legend is off screen. It will be fixed soon

Reply:
Oh, I see. The legend is extending some kind of bounding box behind the scenes?